### PR TITLE
<fix> edit home about and news

### PR DESCRIPTION
**Pull request by Samwa-ua**

### CODE REVIEW

**Review by andrii-dziu — APPROVED** [review]

File: client/src/app/home-page/components/home-about/home-about.component.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <a name="about"></a>
 <section class="about__main">
-  <video autoplay muted loop id="myVideo" poster="../../../../assets/poster.png">
+  <video autoplay muted loop id="myVideo"
+    poster="https://firebasestorage.googleapis.com/v0/b/vetspets.appspot.com/o/assets%2Fposter.png?alt=media&token=24a9989b-b130-4546-8b3f-e1f2c8caeac8">
     <source
-      src="https://firebasestorage.googleapis.com/v0/b/vetspets.appspot.com/o/assets%2Fposter.png?alt=media&token=24a9989b-b130-4546-8b3f-e1f2c8caeac8"
+      src="https://firebasestorage.googleapis.com/v0/b/vetspets.appspot.com/o/1647895783814_video.mp4?alt=media&token=eeded07d-a73b-4931-a2e5-12c1f1cc29b6"
       type="video/mp4">
   </video>
```
Comment (Andriy445) [inline client/src/app/home-page/components/home-about/home-about.component.html:8]: GG VP

File: client/src/app/home-page/components/home-about/home-about.component.html [hunk 2]
```diff
@@ -1,8 +1,9 @@
 <a name="about"></a>
 <section class="about__main">
-  <video autoplay muted loop id="myVideo" poster="../../../../assets/poster.png">
+  <video autoplay muted loop id="myVideo"
+    poster="https://firebasestorage.googleapis.com/v0/b/vetspets.appspot.com/o/assets%2Fposter.png?alt=media&token=24a9989b-b130-4546-8b3f-e1f2c8caeac8">
     <source
-      src="https://firebasestorage.googleapis.com/v0/b/vetspets.appspot.com/o/assets%2Fposter.png?alt=media&token=24a9989b-b130-4546-8b3f-e1f2c8caeac8"
+      src="https://firebasestorage.googleapis.com/v0/b/vetspets.appspot.com/o/1647895783814_video.mp4?alt=media&token=eeded07d-a73b-4931-a2e5-12c1f1cc29b6"
       type="video/mp4">
```
Comment (sanderpa) [inline client/src/app/home-page/components/home-about/home-about.component.html:7]: congratulation!!!!

**Review by sanderpa — APPROVED** [review]

File: client/src/app/home-page/components/home-about/home-about.component.html (hunk 2 above)
Comment (Samwa-ua) [reply client/src/app/home-page/components/home-about/home-about.component.html:7]: nice work!

File: client/src/app/home-page/components/home-about/home-about.component.html (hunk 1 above)
Comment (Samwa-ua) [reply client/src/app/home-page/components/home-about/home-about.component.html:8]: GG

**Pull request merged**